jetty-util/src/main/java/org/eclipse/jetty/util/thread/QueuedThreadPool.java

```diff
@@ -689,6 +689,7 @@ public Thread newThread(Runnable runnable)
         thread.setDaemon(isDaemon());
         thread.setPriority(getThreadsPriority());
         thread.setName(_name + "-" + thread.getId());
+        thread.setContextClassLoader(QueuedThreadPool.class.getClassLoader());
```

Comment:
Do we really want the `QueuedThreadPool` classloader? What if it's a subclass loaded by a different classloader?